examples/map_transformations/autoalign_aia_hmi.py

```diff
@@ -0,0 +1,66 @@
+"""
+==============================
+Auto-Aligning AIA and HMI Data
```

Comment:
I think we should specify in the title that this is just a plotting thing or else it's a little oversold on the examples index.
```suggestion
Auto-Aligning AIA and HMI Data in Plots
```

Reply:
Renamed